cmd/skupper/skupper.go

```diff
@@ -197,6 +197,7 @@ func main() {
 			}
 			if err != nil {
 				fmt.Println(err.Error())
+				os.Exit(1)
```

Comment:
just as a comment, the "log" module has the log.FatalX family that does what you are doing manually here (printing and exiting with error code 1, with the difference that as it is the log module, it also prints the logging header, i.e.:

```
func main() { log.Fatalf("Bye.") }
```
produces:
![image](https://user-images.githubusercontent.com/13121406/87709608-e6c79b00-c77a-11ea-8549-d33407782fb2.png)

Reply:
Yes, however at present we aren't using the log module in the output for the cli. I felt an explicit exit was the cleanest solution for now.

Reply:
Correct!